.prettierrc

```
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2
+}
```

Fastify uses standard across all of its codebases, plugins included. It is nice to have prettier, here, however, I think we should stick to the same rules of standard. The most visible ones are:
```suggestion
  "semi": false,
  "singleQuote": true
}
```

---

Ok added these. I tried switching to both `standardx` and `ts-standard` but the first doesn't work without a lot of extra configuration and the second mandates the use of a lot of annoying TypeScript rules that make for slower code as opposed to stylish code, like explicit casts to booleans for if conditions everywhere. Style wise I totally agree that things should be in line so I added those changes and lint'fixed. If there is a better way or a ... standard ... way to do TS for `fastify/*` repos I am all ears.

---

It would is nice to find a common way of doing this. This is a good starting point, though!